src/site/stages/build/process-cms-exports/index.js

```diff
@@ -158,7 +158,9 @@ const entityAssemblerFactory = contentDir => {
     // Recursively expand entity references
     for (const [key, prop] of Object.entries(filteredEntity)) {
       const isEntityArray =
-        Array.isArray(prop) && prop.some(e => e.target_uuid && e.target_type);
+        Array.isArray(prop) &&
+        prop.some(e => e.target_uuid && e.target_type) &&
+        key !== 'image';
```

Comment:
What problem does this solve?

Reply:
In the `media-image` transformer, we need to get fields `alt`, `title`, `width`, and `height` from the `image` field in the cms export. Because the `image` field also contains `target_type` and `target_uuid` (which map to a file entity) the transformation process performs entity expansion on the `image` field and transforms it to a `file` entity, and we loose the fields we needed from `image`. So I prevented the `image` field from being transformed.

We still have the `thumbnail` field which has the same `target_type` and `target_uuid` as the `image` field, so `thumbnail` still gets transformed to a `file` entity.

You can view `media.d02003a5-0798-4899-b4cd-8689aa4145b9.json` as an example and see the `thumbnail` and `image` fields.